app.js

```diff
@@ -0,0 +1,71 @@
+var playerInput = prompt("Choose your weapon: rock, paper, or scissors. Type EXIT to exit the app.");
+
+// LOOP TO MAKE THE PROMPT REAPPEAR
+while (playerInput !="EXIT") {
+
+  // THE COMPUTER'S RANDOM MOVE
+  var computerMove = Math.random();
+  if (computerMove < 0.34){
+    computerMove = "rock";
+  }
+  else if (computerMove <= 0.67){
+    computerMove = "paper";
+  }
+  else {
+    computerMove = "scissors";
+  }
+
+// COMPARING THE TWO SCORES TO SEE WHO'S WON
+  function checkScore(playerInput, computerMove){
+
+    if (playerInput === computerMove){
+      alert("It's a draw!");
+    }
+    else if (playerInput === "rock" && computerMove === "scissors"){
+      alert("Rock beats scissors: you win a point!");
+      var playerPoint = 1;
```

Comment:
You are defining playerPoint and computerPoint here, not changing their existing value. This means every time a player gains a point, their score is reset to one. You should have defined these variables outside of this function and had this code increment them by one every time they win.